### ISSUE TITLE
Forge?

### ISSUE THREAD

**Issue by Ti0Bi** [Aton-Kish/your-reputation#21]
Any plans to port this mod to Forge 1.19+ (Jade)? If so, that would be great 😃

**Comment by Aton-Kish** [comment]
Unfortunately not. I'm not familiar with Forge mod development.

**Comment by unilock** [comment]
For what it's worth, I managed to get the 1.19.2 Jade version of Your Reputation running on Forge using [Architectury Loom](https://github.com/architectury/architectury-loom).

IIRC, I used Architectury's "1.19.2-forge-fabric-mixin" [template](https://github.com/architectury/architectury-templates/releases/latest), copied the entire source set to the `common` module, copied `atonkish.reputation.ReputationPlugin` and `atonkish.reputation.provider.*` to the `fabric` and `forge` modules verbatim (and deleted them from the `common` module), and adjusted package names and dependencies and such to get it working.

I didn't make a PR, because I think it can be optimized by keeping all Jade-related logic in the `common` module and delegating Jade API calls to their respective Fabric / Forge versions, rather than copying the plugin and providers in their entirety. (Jade doesn't have an intermediary library at the moment, unfortunately.)

I won't be at my computer for the next few hours, but I can upload my fork to GitHub then, if you're interested.
